pkg/resourcecache/main.go

```diff
@@ -33,7 +33,7 @@ type resourceCache struct {
 	log logr.Logger
 }
 
-var KyvernoDefaultInformer = []string{"ConfigMap", "Secret", "Deployment", "MutatingWebhookConfiguration", "ValidatingWebhookConfiguration"}
+var KyvernoDefaultInformer = []string{"ConfigMap", "Secret", "Deployment", "Namespace", "Pod", "Job", "Service", "MutatingWebhookConfiguration", "ValidatingWebhookConfiguration"}
```

Comment:
We don't want to create resource caches for these new resources by default, as they could take too much memory. We should also consider removing configMap and Secret, there was an issue relates to the cache https://github.com/kyverno/kyverno/issues/1540#issuecomment-814076017.

Reply:
will revert back the changes to api call.